caddy/caddy.v

```
@@ -0,0 +1,9 @@
+module caddy
+
+import os
+
+pub fn run_reverse_proxy(domain string, port int) {
```

this should return an error in case of failure e.g port is used

---

yeah, this makes sense, but since this will be called from spawn and spawn has nothing to do with the result and we don't call wait because we need this to run in the background, spawn doesn't show anything if error happened in this case, although print shows the error in the console